packages/blade/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -510,13 +510,15 @@ export const WithValueDisplay = (args: AllDropdownProps): JSX.Element => {
 
   return (
     <BaseBox minHeight={300}>
-      <Text>Selected Values: {dropdownValues}</Text>
+      <Text>Selected Values: {JSON.stringify(dropdownValues)}</Text>
       <BaseBox marginTop="spacing.5" />
       <Dropdown selectionType={selectionType}>
         <SelectInput
           label="Select Action"
-          onChange={({ values }) => {
-            setDropdownValues(values.join(', '));
+          onChange={({ name, values }) => {
```

Comment:
it was working when state was string but it was breaking when state was object. Changed it in the story as well so we catch these issues.